actions/generate.action.ts

```diff
@@ -121,7 +121,10 @@ const mapSchematicOptions = (inputs: Input[]): SchematicOption[] => {
   const options: SchematicOption[] = [];
   inputs.forEach((input) => {
     if (!excludedInputNames.includes(input.name) && input.value !== undefined) {
-      options.push(new SchematicOption(input.name, input.value));
+      const keepInputName = input.options
```

Comment:
what is this `keepInputName` for? can't we have the `--skip-import` flag without adding this logic? :thinking:

Reply:
I don't think it's possible.

https://github.com/nestjs/nest-cli/pull/1580/files/1a0213e2b53689d9672e14d00b7390d2cd9636a7#diff-a9504c3d88b9ec949ea58209a375cf34b9a9772ab092bdb1f0135a16083766c5R81

If an option is passed in, it will be handled by default. So that the skipImport passed to the schema can't match.

https://github.com/nestjs/nest-cli/pull/1580/files/1a0213e2b53689d9672e14d00b7390d2cd9636a7#diff-3621e075da6e7c4276a11edf261e24aa63aae21900c30a2184add622af5f33b1L16

Reply:
got it, thanks. Looks like this is a limitation of that `SchematicOption#toCommandString` method as it expects that long names for flags should be translated to kebab case

Reply:
Yes, `Schematic Option#toCommandString` affects the parameters ultimately passed to `Schema`.